internal/cmd/keys.go

```diff
@@ -157,7 +157,7 @@ func newKeysListCmd(cli *CLI) *cobra.Command {
 				return err
 			}
 
-			keys := &api.ListResponse[api.AccessKey]{}
+			keys := &api.ListResponse[api.AccessKey]{} // nolint
```

Comment:
False positive: was being detected as unused but `keys.Items` is used

Reply:
I don't think this is a false positive. The value of `keys` is never used here, it's being overwritten.

This should change to just a variable declaration:
```suggestion
			var keys *api.ListResponse[api.AccessKey]
```